modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+The `newline_behavior` option has been added to {productname} 6.1 to cater to these use-cases.
+
+`newline_behavior` affects what is added to a document when the `mceInsertNewLine` command is run or when any of the four _create a new line_ keystrokes are pressed: *Return*, *Enter*, *Shift-Return*, or *Shift-Enter*.
+
+By default these keystrokes work as follows:
+
+1. pressing *Return* or *Enter* creates a new paragraph.
++
+For example, in ordinary text, pressing *Return* closes the current `<p>` tag, moves the insertion point in the {productname} editor to a new line and opens a new `<p>` tag.
+   
+2. pressing *Shift-Return* or *Shift-Enter* creates a new line.
++
+For example, in ordinary text, pressing *Shift-Return* enters two `<br>` tags and moves the insertion point in the {productname} editor to a new line.
+
+The `newline_behavior` option takes one of four values:
+
+* `block`
++
+`block` sets {productname} to treat all four keystrokes as if they are *Return*.
+  
+* `linebreak`
++
+`linebreak` sets {productname} to treat all four keystrokes as if they are *Shift-Return*.
+  
+* `invert`
++
+`invert` swaps the behavior or *Return* and *Shift-Return* (and *Enter* and *Shift-Return*). That is, pressing *Return* results in *Shift-Return* behavior, and vice versa.
+
+* `default`
++
+`default`, as its name suggests, sets {productname} to do what it does by default when *Return*, *Enter*, *Shift-Return*, or *Shift-Enter* are pressed.
+
+NOTE: Two other commands that, in their effect, add new lines to a {productname} document are `insertParagraph` and `InsertLineBreak`. Their behavior is not affected by any `newline_behavior` setting.
+
+
+=== New `transparent` property for `iframe` dialog component
+
+As part of the fix for correcting the presentation of {productname} content in the *Preview* and *Insert template* dialogs, the `iframe` dialog component now supports a new `transparent` property.
+
+
+=== New `removeAttributeFilter` and `removeNodeFilter` functions added to the DomParser and DOM Serializer APIs
+
+Two new filter functions — `removeAttributeFilter` and `removeNodeFilter` — have been added to the `DomParser` and DOM `Serializer` APIs.
+
+Each function takes two arguments: `name` and `callback`.
+
+`name` is required.
+
+`callback` is optional.
+
+`name` can take a comma-separated list of attribute or node names.
+
+If both `name` and `callback` are provided, the functions remove the specified callback.
+
+If only `name` is provided, the functions remove all callbacks for the specified name.
+
+When only `name` is provided, the new functions work in equivalent fashion to the `editor.off` API, which removes all event handlers when only a name is provided.
+
+//// An example would be a good thing. Also for the updated docs proper.
+
+
+=== New `iframe_template_callback` option in the media plugin.
+
+The new `iframe_template_callback` option, provided by https://github.com/Namstel[Namstel], is an addition to the media plugin.
```

Comment:
We don't normally like to link like this as it can be considered rude and lead to spam being sent, so we should remove this link.
```suggestion
The new `iframe_template_callback` option, provided by Namstel, is an addition to the media plugin.
```